save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/demo/diktat/DiktatDemoMode.kt

```diff
@@ -0,0 +1,17 @@
+package com.saveourtool.save.demo.diktat
+
+/**
+ * Enum that represents possible demo modes in diktat-demo
+ */
+enum class DiktatDemoMode {
```

Comment:
Can we create some common interface, may be? From which there will be inherited our particular cases, like diktat and so on?

Reply:
For now it's hard for me to set the limits on Demo's inputs. Some tools might include run mode and so on, some might not. Not sure we should try to unify it right now. Probably should be done in phase 3

Reply:
Will add `DemoAdditionalParams` as an argument of `runDemo` function

Reply:
Done